script/int-log-compiler.sh

```diff
@@ -157,20 +157,43 @@ TEST_NAME=$(basename "${TEST_BIN}")
 SIM_LOG_FILE=${TEST_BIN}_simulator.log
 SIM_PID_FILE=${TEST_BIN}_simulator.pid
 SIM_TMP_DIR=$(mktemp --directory --tmpdir=/tmp tpm_server_XXXXXX)
-for i in $(seq 5); do
-    SIM_PORT=$(od -A n -N 2 -t u2 /dev/urandom | awk -v min=1024 -v max=65535 '{print ($1 % (max - min)) + min}')
-    simulator_start ${SIM_BIN} ${SIM_PORT} ${SIM_LOG_FILE} ${SIM_PID_FILE} ${SIM_TMP_DIR}
-    if [ $? -eq 0 ]; then
-        break;
-    else
-        echo "Failed to start simulator on port ${SIM_PORT}, retrying"
+PORT_MIN=1024
+PORT_MAX=65534
+BACKOFF_FACTOR=2
+BACKOFF=1
+for i in $(seq 10); do
```

Comment:
Do we really need to wait that long? When all probes fail then it will take ~16 minutes to run. I would do (seq 6) max which will make it ~1 minute max.

Reply:
Reasonable. I just never did the math to see how long the maximum wait time would be. 10 was just a "nice round number" :smile: A minute sounds reasonable.